.github/workflows/pr-build-live-branch.yml

```diff
@@ -38,7 +38,7 @@ jobs:
           PR_NUMBER: ${{ github.event.number }}
           PR_HEAD_SHA: ${{ github.event.pull_request.head.sha }}
         run: |
-          DEV_SUFFIX_VERSION="dev_PR-${PR_NUMBER}_commit-${PR_HEAD_SHA}"
+          DEV_SUFFIX_VERSION="dev_PR-${PR_NUMBER}_commit-${PR_HEAD_SHA:0:7}"
```

Comment:
A small change to display [the shorten commit hash](https://user-images.githubusercontent.com/10045087/226828532-c0dc372a-39b3-43e5-924d-f42977f520e1.png) rather than [the full commit hash string](https://user-images.githubusercontent.com/10045087/226828410-b87b574f-5871-4575-a87a-a3f2982476b0.png) in Jetpack Beta UI.